docs/python/examples/plot_dl_keras.py

```diff
@@ -85,7 +84,7 @@
 # Let's get more comprehensive results.
 
 if ok:
-    from keras.applications.densenet import decode_predictions
+    from tensorflow.keras.applications.densenet import decode_predictions
```

Comment:
Minor nit pick but this does not seem to be related to the underscore changes ...

Reply:
No it is related to the documentation. I can remove it.